api/machinemanager/machinemanager.go

```diff
@@ -81,3 +81,21 @@ func (client *Client) destroyMachines(method string, machines []string) ([]param
 	}
 	return allResults, nil
 }
+
+// UpdateMachineSeries updates the series of the machine in the db.
+func (client *Client) UpdateMachineSeries(machineName, series string, force bool) error {
+	args := params.MachineUpdateSeriesArgs{
+		Args: []params.MachineUpdateSeriesArg{{
+			MachineName: machineName,
```

Comment:
This should be machine tag.
It could be argued that the ApplicationUpdateSeriesArg struct should also take a tag, and there be a single struct used for both machine and application series updates; this struct would have an entity tag param. The fact that ApplicationUpdateSeriesArg takes an application name is reasonable as we use app name elsewhere but we should try to 1. minimise the number of bespoke structs, 2. use tags where possible